src/Components/Main/Pages/UsersPage/UsersPage.js

```diff
@@ -13,32 +14,36 @@ export const UsersPage = ({
                             setPage,
                             setUsers,
                             setTotalCount,
-                            isFetching,
-                            toggleIsFetching
+                            isFetchingUsers,
+                            toggleIsFetchingUsers
                           }) => {
+  const mounted = useMounted();
```

Comment:
The same applies here =)
It makes more sense to add mounted hook into parent component

Reply:
Done!